src/app/core/store/customer/user/user.actions.ts

```diff
@@ -25,6 +25,12 @@ export const loadCompanyUserSuccess = createAction('[User API] Load Company User
 
 export const logoutUser = createAction('[User] Logout User');
 
+export const revokeApiToken = createAction('[User] Revoke Api Token');
```

Comment:
introduce logoutUserSuccess and logoutUserFail actions,  use [User API] in brackets

remove revokeApiToken actions